### PR TITLE
Improve the documentation for releasing a new version.

### PR DESCRIPTION
**Pull request by codeviking**
This makes several improvements to the documentation per gaps I noticed
while working through the process today with @aaronsarnat.

### CODE REVIEW

**Comment by aaronsarnat** [comment]
@codeviking as soon as you merge this, I will publish a `6.0.1` release that adds this and fixes the react-router bug.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -32,32 +32,66 @@ and [yarn](https://yarnpkg.com/lang/en/).
...
+   Note, if you haven't been added to the `@allenai` NPM organization you'll
+   get a permissions error when you attempt the `publish` command below. If
+   this occurs ask someone from [REVIZ](mailto:reviz@allenai.org) to add you.
+
+4. Create a new version, we intend to follow [semantic versioning](https://semver.org/),
+   though at the moment we're pre-1.0, and accordingly increase the `minor` version
+   for significant changes, and the `patch` version for small ones.
 
    ```bash
-   npm version major|minor|patch
```
Comment (jonborchardt) [inline CONTRIBUTING.md]: why are we nolonger suggesting this?
this seems correct.

Comment (codeviking) [reply CONTRIBUTING.md]: In walking through it, @aaronsarnat wasn't clear that it gets replaced by the actual version numbers.

I just replaced it with:

```shell
# Replace `0.5.8` with the version you're releasing
npm version 0.5.8
```

Which I think makes it a bit more clear?

Comment (jonborchardt) [reply CONTRIBUTING.md]: er.. ok, i like the old way ;-P

**Pull request merged**